cpmpy/solvers/exact.py

```diff
@@ -127,7 +127,7 @@ def _fillVars(self):
         if not self.xct_solver.hasSolution():
             self.objective_value_ = None
```

Comment:
yes, here we do empty the objective value, so we should do it also in all solvers

Reply:
It is actually already there for all solvers, right before the translation of the solution